### PR TITLE
New profile for Creality Ender 3 V3 KE

### PR DESCRIPTION
**Pull request by bistory**
This PR adds a new profile for Creality Ender 3 V3 KE.
Previous PR (#3192) was automatically closed, so I open this one...

It hasn't been tested (yet), however, I have the machine so I will fine-tune the profiles if needed.
Basically, it's a Ender 3 V3 SE with a better hotend, a linear rail for X axis and running on Klipper...

### CODE REVIEW

File: resources/profiles/Creality/filament/Creality Generic PLA High Speed @Ender-3V3-all.json [hunk 1]
```diff
@@ -7,8 +7,5 @@
     "inherits": "Creality Generic PLA @Ender-3V3-all",
     "filament_max_volumetric_speed": [
         "23"
-    ],
-    "compatible_printers": [
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic PLA High Speed @Ender-3V3-all.json:11]: Due to some code issue, we need to specify compatible_printers in all instantiatiable profiles for now.
So, for now let's maintain/update this property

File: resources/profiles/Creality/filament/Creality Generic PLA Matte @Ender-3V3-all.json [hunk 2]
```diff
@@ -5,8 +5,5 @@
 	"from": "system",
 	"instantiation": "true",
 	"inherits": "Creality Generic PLA @Ender-3V3-all",
-	"filament_max_volumetric_speed": ["18"],
-	"compatible_printers": [
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic PLA Matte @Ender-3V3-all.json:9]: same

File: resources/profiles/Creality/filament/Creality Generic PLA Silk @Ender-3V3-all.json [hunk 3]
```diff
@@ -5,8 +5,5 @@
 	"from": "system",
 	"instantiation": "true",
 	"inherits": "Creality Generic PLA @Ender-3V3-all",
-	"filament_max_volumetric_speed": ["7.5"],
-	"compatible_printers": [
-		"Creality Ender-3 V3 SE 0.4 nozzle"
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic PLA Silk @Ender-3V3-all.json:10]: same

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by bistory** [comment]
Thanks, it's done ;-)

**Review by bistory — COMMENTED** [review]
Reverted code simplification

**Comment by SoftFever** [comment]
Thank you for the quick response.
Looks good now.

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
@bistory 
I found one more thing we missed for V3 SE/KE,
The default materials are not updated which will cause first time user not see proper materials.
We need to update them as well
```
"default_materials": "Creality Generic PLA;Creality Generic PETG;Creality Generic ABS"
```
in file:
resources/profiles/Creality/machine/Creality Ender-3 V3 SE.json
resources/profiles/Creality/machine/Creality Ender-3 V3 KE.json

**Comment by bistory** [comment]
> @bistory I found one more thing we missed for V3 SE/KE, The default materials are not updated which will cause first time user not see proper materials. We need to update them as well
> 
> ```
> "default_materials": "Creality Generic PLA;Creality Generic PETG;Creality Generic ABS"
> ```
> 
> in file: resources/profiles/Creality/machine/Creality Ender-3 V3 SE.json resources/profiles/Creality/machine/Creality Ender-3 V3 KE.json

Thanks for the feedback I fixed it, do you need a new PR ?

**Comment by SoftFever** [comment]
> > @bistory I found one more thing we missed for V3 SE/KE, The default materials are not updated which will cause first time user not see proper materials. We need to update them as well
> > ```
> > "default_materials": "Creality Generic PLA;Creality Generic PETG;Creality Generic ABS"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > in file: resources/profiles/Creality/machine/Creality Ender-3 V3 SE.json resources/profiles/Creality/machine/Creality Ender-3 V3 KE.json
> 
> Thanks for the feedback I fixed it, do you need a new PR ?

👍 
Yes, please.

**Comment by Alexgrau88** [comment]
Hello guys, im new to 3d print world, curios about one issue I'm facing with a Ender3V3KE, installed OrcaSlicer_Windows_Installer_1.8.1.exe , but I'm not saying the printer profile, am I missing something? I see that pull 3216 is merged, is it not committed to V 1.8.1 ?

**Comment by bistory** [comment]
> Hello guys, im new to 3d print world, curios about one issue I'm facing with a Ender3V3KE, installed OrcaSlicer_Windows_Installer_1.8.1.exe , but I'm not saying the printer profile, am I missing something? I see that pull 3216 is merged, is it not committed to V 1.8.1 ?

V1.8.1 is 3 weeks old, this PR is one week old ;-)
This PR is already available in v1.9.0-alpha and improvements are in development on my side, I plan to make a new PR as soon as possible before v1.9.0 final release :-)

**Comment by Alexgrau88** [comment]
> > Hello guys, im new to 3d print world, curios about one issue I'm facing with a Ender3V3KE, installed OrcaSlicer_Windows_Installer_1.8.1.exe , but I'm not saying the printer profile, am I missing something? I see that pull 3216 is merged, is it not committed to V 1.8.1 ?
> 
> V1.8.1 is 3 weeks old, this PR is one week old ;-) This PR is already available in v1.9.0-alpha and improvements are in development on my side, I plan to make a new PR as soon as possible before v1.9.0 final release :-)

Yes indeed, i tried the 1.9.0 and its there. Thanks for the reply

**Review by esn814 — COMMENTED** [review]
Nice work ppl

**Pull request merged**